packages/miniflare/src/plugins/kv/index.ts

```diff
@@ -96,9 +99,23 @@ export const KV_PLUGIN: Plugin<
 	}) {
 		const persist = sharedOptions.kvPersist;
 		const namespaces = namespaceEntries(options.kvNamespaces);
+		const KV_OBJECT_SERVICE_NAME = addPersistPathSuffix(
+			`${KV_PLUGIN_NAME}:object`,
+			persist
+		);
+		const KV_STORAGE_SERVICE_NAME = addPersistPathSuffix(
```

Comment:
Won't changing the persisPath break compatibility with existing local kv namespaces?

Reply:
I don't think so. This doesn't change the service name of the namespace service but just the underlying object and storage service.

Reply:
Do you mind performing an hand-over-hand test where you write data into a namespace with the current wrangler build, and then try to read it with this version? I'm afraid this might change the underlying storage path, and data becomes unaccessible between versions. If that's the case, this might require a major bump.

Reply:
Sure. Happy to give it a manual test to confirm the behavior. Definitely don't want to break existing users because of it. I will update here after the test.

Reply:
@pmiguel Here is an example app that saves a random number in a KV namespace: https://github.com/edmundhung/kv-test

You can then update it to the pre-release version by running 
```sh
npm install --save-dev https://prerelease-registry.devprod.cloudflare.dev/workers-sdk/runs/14644497405/npm-package-wrangler-9026
```

The same number should be shown.